shop/src/main/resources/application.properties

```diff
@@ -1 +1,20 @@
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.properties.hibernate.default_schema=shop
 
+spring.jpa.hibernate.ddl-auto=none
```

Comment:
You should use the value validate here. https://docs.jboss.org/hibernate/orm/6.2/userguide/html_single/Hibernate_User_Guide.html#configurations-hbmddl